pkg/operator/operator.go

```diff
@@ -116,9 +116,9 @@ func (rt *ReconciliationTracker) getStatus(k string) (ReconciliationStatus, bool
 	return s, true
 }
 
-// GetCondition returns a monitoringv1.Condition for the last-known
+// GetReconciledCondition returns a monitoringv1.Condition for the last-known
 // reconciliation status of the given object.
-func (rt *ReconciliationTracker) GetCondition(k string, gen int64) monitoringv1.Condition {
+func (rt *ReconciliationTracker) GetReconciledCondition(k string, gen int64) monitoringv1.Condition {
```

Comment:
IIRC I omitted `reconciled` in the function name because the struct is already named `ReconciliationTracker`.